.python-version

```diff
@@ -1 +1 @@
-3.5.6
```

Comment:
This is only used by PyEnv, which can install the more modern `3.6.11` just as well.